jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -2720,9 +2720,17 @@ public String toLowerCase(Locale locale) {
 		}
 
 		if (StrHWAvailable() && language == "en") { //$NON-NLS-1$
```

Comment:
Since we are in this area of code can we rename this method here to something more sane and place it in the JitHelpers class with proper documentation? Perhaps `supportsIntrinsicCaseConversion` or if you can think of a better name I'm fine with that?